app/controllers/provider_foreman_controller.rb

```diff
@@ -376,6 +381,14 @@ def cs_filter_tree_rec
     end
   end
 
+  def cm_job_templates_tree_rec
+    nodes = x_node.split('-')
+    type, _id = x_node.split("_").last.split("-")
```

Comment:
I know I've seen this code multiple times.  Can't a method be created to return the parts?